t/lib/Test/Builder/Module.pm

```diff
@@ -3,9 +3,9 @@ package Test::Builder::Module;
 use Test::Builder;
 
 require Exporter;
-@ISA = qw(Exporter);
+our @ISA = qw(Exporter);
 
-$VERSION = '0.03';
+our $VERSION = '0.03';
 
 use strict;
```

Comment:
Not the end of the world but this strict should probably move up.

Reply:
Is this not one of the files that would be removed in your p.r. https://github.com/Perl-Toolchain-Gang/ExtUtils-Install/pull/25?

Reply:
It is.

Reply:
#25 now merged, this is sorted :-)